src/Cronofy/UpsertEventRequestBuilder.cs

```diff
@@ -578,6 +589,19 @@ public UpsertEventRequestBuilder RemoveAttendee(string email, string displayName
             return this;
         }
 
+        /// <summary>
+        /// Sets the event as having no start or end.
+        /// </summary>
+        /// <returns>
+        /// A reference to the modified builder.
+        /// </returns>
+        public UpsertEventRequestBuilder NoStartOrEnd()
```

Comment:
Feels a bit nasty doing this, wonder if we can solve this through inheritance more cleanly